resources/js/index.js

```diff
@@ -0,0 +1,20 @@
+const menu = document.getElementById('menuDiv');
+const menuElement = [];
+
+function navigationMenu() {
```

Comment:
It works, but it somehow seems useless to load the client browser with creating DOM for a static navigation menu. There is not much code duplication you win in this way. You might consider utilizing Blade and Blade Components to solve this.